product_replenishment_cost/models/product_supplierinfo.py

```diff
@@ -23,6 +26,11 @@ class ProductSupplierinfo(models.Model):
         help="Net Price",
     )
 
+    def write(self, vals):
+        if vals.get('price'):
```

Comment:
`and not vals.get(last_date_price_updated)`

Reply:
@jjscarafia review solved